packages/wasm/crate/src/tx.rs

```diff
@@ -297,37 +297,48 @@ pub async fn transaction_info_inner(
         };
         match action_view {
             ActionView::Spend(SpendView::Visible { note, .. }) => {
-                let address = note.address();
```

Comment:
More changes due to ownership issues.

Also, note that `address_views` used to be a `BTreeMap` of addresses to address views. Now, it's a map of addresses-encoded-to-byte-arrays to address views. That's because [a recent change](https://github.com/penumbra-zone/penumbra/pull/4285/files#diff-bb4c349bac5ab2231b6774ebbfdaabe48a985d24599f872525f90916a3e9fb76R37) to `Address` resulted in `Address` structs being mutable, since they hold a property of type `OnceLock<...>`. Clippy complains about the use of mutable keys, since their serialization can change over time when they're mutable, thus destroying the integrity of the key.

To work around this, @aubrika and I realized we could use the byte-encoded addresses as keys, rather than the `Address` objects themselves.